test/e2e/util/util.go

```diff
@@ -370,8 +370,11 @@ func EnsureNoCrashingPods(t *testing.T, ctx context.Context, client crclient.Cli
 			}
 
 			for _, containerStatus := range pod.Status.ContainerStatuses {
-				if containerStatus.RestartCount > 0 {
-					t.Errorf("Container %s in pod %s has a restartCount > 0 (%d)", containerStatus.Name, pod.Name, containerStatus.RestartCount)
+				// Setting this to 5 just to cover external controllers and operators that
+				// need some tries to run properly until other components are accesible
+				// E.G csi-snapshot-controller
```

Comment:
We need to fix crash loops.
If we wanted to temporary relax, let's put a PR for that aside from this one.

Reply:
Agree, is a temporary measure, I will change before the last Review ;)